docs/launchpad/quick-start.md

```diff
@@ -78,14 +78,28 @@ Remember that each cluster might have specific setup steps or requirements, espe
 - [x] We have installed all the tooling dependencies on our local machine, which will be used to control the cluster
 - [ ] Next: Copy `sample.helmfile.yaml` to `helmfile.yaml` and edit it to customize and deploy Namespaces on the Kubernetes cluster
 
-### Customize your helmfile.yaml
+### Customize your helmfiles
+
+To get started with Helmfile, if you don’t already have a `helmfile.yaml`, you can begin by copying the provided sample configuration file named `sample.helmfile.yaml`:
 
-If you don't have an `helmfile.yaml` file yet, you can start with copying the existing sample one (`sample.helmfile.yaml`):
 ```shell
 cp sample.helmfile.yaml helmfile.yaml
 ```
 
-and edit it with your editor of choice. Under that file, there is an `helmfiles:` section declaring Namespaces to be deployed on the cluster, looking like this:
+After copying, open `helmfile.yaml` in your preferred text editor to make necessary modifications. Within this file, you will find a `helmfiles:` section which organizes deployment configurations by namespace through multiple helmfile paths:
+
+```yaml
+helmfiles:
```

Comment:
I think we may want to add the graph namespace, as the quickstart initially was to go from 0 to working Graph-Node. Or do we want to change that?